src/combination.ts

```diff
@@ -101,3 +124,21 @@ export const combineQueries = <Queries extends QueryObserverResult[]>(
     queries: queries as any,
   };
 };
+
+export const suspenseCombineQueries = <Queries extends QueryObserverResult[]>(
+  queries: [...Queries],
+): SuspenseCombinedQueriesResult<Queries> => {
+  const base = getBase(queries);
+
+  // Loading and Error states will be handled by suspense and error
+  // boundaries so unlike the non-suspense version we only need to
+  // account for the DefinedQueryObserverResult state
+  return {
+    ...base,
+    status: 'success',
+    isPending: false,
+    data: queries.map((query) => query.data) as any,
+    isError: false,
+    queries: queries as any,
```

Comment:
Why did we need to cast as any here?

Reply:
I am not exactly sure and there are other places we are doing this in this same file - I went ahead and added what I could in a comment from taking a closer look